src/networking/protocol.cpp

```diff
@@ -721,7 +734,7 @@ bool PacketManager::ManageStartupPacket() {
     return false;
   }
   status = ProcessStartupPacket(&pkt, responses);
-  if (!WritePackets(responses, &client_) || !status) {
+  if (!WritePackets(responses, &client_, true) || status == false) {
```

Comment:
Pass the third boolean argument of WirePackets as default argument for better readability?